testProjects/default/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "handler.js",
   "dependencies": {
-    "iopipe": "0.2.0",
+    "iopipe": "1.5.0",
```

Comment:
iopipe => @iopipe/core ?

Reply:
No, we still want to allow this plugin to work with the `iopipe` lib so it's ok if this uses an older pattern for testing.